st3/sublime_lib/resource_path.py

```diff
@@ -75,11 +101,23 @@ def from_file_path(cls, file_path):
         for root, base in get_resource_roots().items():
             try:
                 rel = file_path.relative_to(base)
+            except ValueError:
+                pass
+            else:
                 return cls(root, *rel.parts)
+
+        for base in get_installed_resource_roots():
+            try:
+                package, *rest = file_path.relative_to(base).parts
             except ValueError:
                 pass
+            else:
+                package_path = cls('Packages', package)
+                if package_path.suffix != '.sublime-package':
+                    continue
+                return package_path.with_suffix('').joinpath(*rest)
```

Comment:
The condition should definitely be inverted. It's probably best to use the ResourcePath functionality unless we specifically want different behavior in the corner cases it covers. For example, how does Sublime deal with a zipfile `Installed Packages/.sublime-package`? I'll take a look later tonight.

Reply:
Yeah, I guess inverting the condition is good enough and what iffed (I so wish this was a word) me the most. Using a ResourcePath for checking and removing the suffix better shows the intention.

I don't think we need to care about these edge cases since they will practically never come up in real scenarios.

Reply:
Fixed the condition.